README.md

```diff
@@ -28,6 +28,14 @@ $ npm install
 ## Download data
```

Comment:
Now there are two ways of downloading data: (1) manually from Metro Extracts, OSM, or Geofabrik or (2) via specifying URL in config and running `npm run download`. 

For the first option, the `openstreetmap.download` property is irrelevant. And you don't need to run `npm run download`.

For the latter method, it makes more sense to see the "Configuration" section before running `$ PELIAS_CONFIG=<path-to-config> npm run download`. `imports.openstreetmap.datapath` specifies the directory which the pbf file _will be_ downloaded to.

I think the instructions would be clearer if "Download Data" split into the "Manually" and "Using Download Script" and each included the specific steps and example config file. Or, the sections can remain as is but with notes on which step is necessary for which option.